DEVELOPER.md

```diff
@@ -1,188 +1,87 @@
 - [Developer quickstart](#developer-quickstart)
-  - [Code style](#code-style)
-  - [Backend setup](#backend-setup)
-  - [Frontend setup](#frontend-setup)
-  - [Setup using Makefile](#setup-using-makefile)
+  - [Running the project](#running-the-project)
+  - [Running in Docker](#running-in-docker)
+    - [`COMPOSE_PROFILES`](#compose_profiles)
+  - [Useful `make` commands](#useful-make-commands)
+  - [Setting environment variables](#setting-environment-variables)
   - [Slack application setup](#slack-application-setup)
   - [Update drone build](#update-drone-build)
-- [Troubleshooting](#troubleshooting)
-  - [ld: library not found for -lssl](#ld-library-not-found-for--lssl)
-  - [Could not build wheels for cryptography which use PEP 517 and cannot be installed directly](#could-not-build-wheels-for-cryptography-which-use-pep-517-and-cannot-be-installed-directly)
-  - [django.db.utils.OperationalError: (1366, "Incorrect string value ...")](#djangodbutilsoperationalerror-1366-incorrect-string-value-)
-  - [Empty queryset when filtering against datetime field](#empty-queryset-when-filtering-against-datetime-field)
-- [Hints](#hints)
-  - [Building the all-in-one docker container](#building-the-all-in-one-docker-container)
-  - [Running Grafana with plugin (frontend) folder mounted for dev purposes](#running-grafana-with-plugin-frontend-folder-mounted-for-dev-purposes)
-  - [How to recreate the local database](#recreating-the-local-database)
-  - [Running tests locally](#running-tests-locally)
 - [IDE Specific Instructions](#ide-specific-instructions)
   - [PyCharm](#pycharm)
 
 ## Developer quickstart
 
 Related: [How to develop integrations](/engine/config_integrations/README.md)
 
-### Code style
+### Running the project
 
-- [isort](https://github.com/PyCQA/isort), [black](https://github.com/psf/black) and [flake8](https://github.com/PyCQA/flake8) are used to format backend code
-- [eslint](https://eslint.org) and [stylelint](https://stylelint.io) are used to format frontend code
-- To run formatters and linters on all files: `pre-commit run --all-files`
-- To install pre-commit hooks: `pre-commit install`
+1. Firstly, ensure that you have `docker` [installed](https://docs.docker.com/get-docker/) and running on your machine.
+2. Run `make start`. By default this will run everything in Docker, using SQLite as the database and Redis as the message broker/cache. See [Running in Docker](#running-in-docker) below for more details on how to swap out/disable which components are run in Docker.
+3. Open Grafana in a browser [here](http://localhost:3000/plugins/grafana-oncall-app) (login: `oncall`, password: `oncall`).
+4. You should now see the OnCall plugin configuration page. Fill out the configuration options as follows:
```

Comment:
you no longer need to restart your `grafana` container for it to recognize local OnCall. This further simplifies these instructions a bit.